### PR TITLE
AT_004.001 |Main page >Verify the temperature can be switched from Imperial to Metric

### PR DESCRIPTION
**Pull request by Sana291**
https://trello.com/c/aHM7WLak/93-at004001-main-page-verify-the-temperature-can-be-switched-from-imperial-to-metric

### CODE REVIEW

**Review by TimothyHuxx — COMMENTED** [review]
There must be an empty line between each test (it)

File: cypress/e2e/group_javascript_for_qas.cy.js [hunk 1]
```diff
@@ -4,4 +4,10 @@ describe('group_javascript_for_qas', () => {
     it('AT_028.001 | Footer > About us > Verify "About us" link redirects to the corresponding page', function () {
         cy.visit('https://openweathermap.org/');
     });
+    
+    it('AT_004.001 | Main page > Verify the temperature can be switched from Imperial to Metric', function(){
+        cy.visit('https://openweathermap.org');
+        cy.get('.switch-container > div:nth-of-type(3)').click();
+        cy.get('.switch-container > div:nth-of-type(2)').should('contain', 'Metric: °C, m/s').click();
```
Comment (elviraKh1) [inline cypress/e2e/group_javascript_for_qas.cy.js:11]: Ваш тест не соответствует вашему TC... код не проверяет , что произошло переключение

Comment (Sana291) [reply cypress/e2e/group_javascript_for_qas.cy.js:11]: can you explain? part of my AC not to confirm its highlighted, its only # 2. I do have insertion which confirms thats that exactly what is containing when i click.

Comment (elviraKh1) [reply cypress/e2e/group_javascript_for_qas.cy.js:11]: I can't see that the previous temperature was Imperial.

**Comment by Sana291** [comment]
Unfortunately my internet is off , but I see your point. I will work on that , trying to see if I can somehow get different perspective on it as well  as soon as I can get to run test and git Thank you Sent from my iPhoneOn Nov 12, 2022, at 17:33, elviraKh1 ***@***.***> wrote:﻿
@elviraKh1 commented on this pull request.



In cypress/e2e/group_javascript_for_qas.cy.js:
> @@ -4,4 +4,10 @@ describe('group_javascript_for_qas', () => {
     it('AT_028.001 | Footer > About us > Verify "About us" link redirects to the corresponding page', function () {
         cy.visit('https://openweathermap.org/');
     });
+    
+    it('AT_004.001 | Main page > Verify the temperature can be switched from Imperial to Metric', function(){
+        cy.visit('https://openweathermap.org');
+        cy.get('.switch-container > div:nth-of-type(3)').click();
+        cy.get('.switch-container > div:nth-of-type(2)').should('contain', 'Metric: °C, m/s').click();

I can't see that the previous temperature was Imperial.

**Review by elviraKh1 — APPROVED** [review]

**Review by IrynaRo — APPROVED** [review]

**Pull request merged**